lnst/Agent/conditions/WaitForConditionModule.py

```diff
@@ -0,0 +1,49 @@
+import time
```

Comment:
IMO, this shouldn't be part of the `lnst.Agent` package, the main reason is that you define the instance of the class on the controller. Theoretically, lnst packages are designed in a way that the controller machine/environment doesn't have the Agent package available at all. This means that you wouldn't be able to properly import the class on the controller.

IMO this should be in one of:
* the `lnst.Common` package
* the `lnst.RecipeCommon` package
* a new `lnst.<some new package>` that is universal for both the agent and the controller installations (similar to how the `lnst.Devices` package exists), but imo this isn't correct for this case.